tasks/main.yml

```diff
@@ -1,5 +1,10 @@
 ---
-
+- name: Ensure mandatory variables are defined
+  assert:
+    that:
+      - roundcube_mysql_password_enc is defined
```

Comment:
I think it would make more sense to check for `roundcube_mysql_password` instead, no? `roundcube_mysql_password_enc` is only our default vault variable, but people can set `roundcube_mysql_password` otherwise if they want.

Reply:
Agree